### ISSUE TITLE
Add support for quilt

### ISSUE THREAD

**Issue by yuri0r** [froehlichA/pax#80]
there is a seeming influx of mods moving from fabric to [quilt](https://quiltmc.org/en/about/faq-devs/).

i can't maintain packs without pax anymore so i'd really apreciate adding support.

**Comment by Trinsdar** [comment]
Lol no there is not,  there are at most 100 quilt exclusive mods, the rest of them are just fabric mods marked quilt compatible

**Comment by unix-supremacist** [comment]
i'd strongly suggest against this for a few reasons

1. quilt as it stands has very few mods, you'd be more effective working on liteloader or modrinth support
2. support for quilt can be seen as an endorsement of quilt, which i'm not going to get into the drama. but i know tons of people quilt has hurt directly or indirectly
3. quilt it self doesn't have the manpower to last, that's why so much was taken from upstream fabric, and why they are trying to replace stuff like intermediary with something they don't have to have as much manpower to work on.

so TLDR quilt support at this stage is a gamble, and will probably just lead to dead code

**Comment by froehlichA** [comment]
My last commit had to fix Quilt mods being selected for Forge anyway, so no point putting it off - And supporting Quilt modpacks & mods fully is simple enough to do :P

Support is implemented in the [latest release](https://github.com/froehlichA/pax/releases/tag/v1.10.0). Since not many people use Quilt with PAX according to my knowledge, please let me know if there's issues.
